kobo/apps/mfa/models.py

```diff
@@ -9,6 +10,27 @@
 )
 
 
+class KoboMFAPerUserActivation(models.Model):
+
+    class Meta:
+        verbose_name = 'Per-user activation'
+        verbose_name_plural = 'Per-user activations'
```

Comment:
i think `verbose_name` is supposed to be lowercase, and Django will capitalize it as needed

Reply:
Oh. I did not know that ;-) thx.

Reply:
For the record, it's also true for `verbose_name_plural`